### PR TITLE
add cinema-booking pre-review

### PR DESCRIPTION
**Pull request by Roophee**
### Html Movie Seat Booking

[Demo](https://roophee.github.io/cinema_project/)
[Code base](https://github.com/kottans/frontend-2021-classroom/tree/cinema-project-roophee/submissions/Roophee/html-movie-seat-booking
)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Roophee/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+                    <input type="text" placeholder="Поиск" class="header__search-form-input" name="query">
+                </form>
+            </div>
+            <div class="header__account">
+                    <div class="header__account-avatar">
+                        <img class="header__account-avatar-img" src="img/user_avatar3.png" alt="avatar">
+                    </div>
+                    <div class="header__account-login-btn">
+                        Login
+                    </div>
```
Comment (A-Ostrovnyy) [inline submissions/Roophee/html-movie-seat-booking/index.html]: Use interactive elements for elements like this.

File: submissions/Roophee/html-movie-seat-booking/style.scss [hunk 2]
```diff
@@ -0,0 +1,670 @@
...
+    }
+}
+
+.table {
+
+    &-border{
+        color: $font-color;
+        border-bottom: 1px solid $lineColor;
+        margin: 0;
+    }
```
Comment (A-Ostrovnyy) [inline submissions/Roophee/html-movie-seat-booking/style.scss]: Apply these styles to the `tr` once, reduce using similar classes.

File: submissions/Roophee/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+                        <td class="table-data table-border">Заместителя шерифа округа Керн Джо Дикона по прозвищу Дики отправляют в Лос-Анджелес – обычная командировка для сбора улик. Но там его привлекают к поискам серийного убийцы, который терроризирует город. Охоту на маньяка возглавляет Джим Бэкстер, сержант департамента шерифа Лос-Анджелеса. Его впечатляет полицейское чутье Дики, и он неофициально привлекает его к расследованию. Они всё ближе к цели, но Бэкстер не догадывается, что тем самым они воскрешают призраков прошлого Дики, раскрывая такие тайны, которые могут всколыхнуть куда более темные воды, чем его нынешнее дело.</td>
+                    </tr>
+                </tbody></table>
+            </article>
+        </section>
+        <section class="content__video">
+            <div class="content__video-label">
+                <P>
+                    Смотреть трейлер
+                </P>
```
Comment (A-Ostrovnyy) [inline submissions/Roophee/html-movie-seat-booking/index.html]: Same as at line 47.

File: submissions/Roophee/html-movie-seat-booking/index.html [hunk 4]
```diff
@@ -0,0 +1,428 @@
...
+                                </tr>
+                                <tr>
+                                    <th class="seats__body-table-row">Ряд</th>
+                                    <td class="seats__body-table-seats weight400">Место
+                                    </td>
+                                    </tr>
+                            <tr>
+                            <th class="seats__body-table-row">1</th>
+                            <td class="seats__body-table-seats">
+                                <input class="seats__body-table-seats-place" type="checkbox" id="r1_s1" value="r1_s1" name="ticketSeat"><label class="whitespaces" for="r1_s1">1</label>
```
Comment (A-Ostrovnyy) [inline submissions/Roophee/html-movie-seat-booking/index.html:145]: Add `cursor: pointer` to all interactive elemnts.

File: submissions/Roophee/html-movie-seat-booking/index.html [hunk 5]
```diff
@@ -0,0 +1,428 @@
...
+                            Pepsi 0,5 (стакан)
+                        </label>
+                        <input type="checkbox" name="ticketCustom" id="3d_glasses" value="3d">
+                        <label for="3d_glasses">
+                            3D-очки
+                        </label>
+                    </div>
+                    <div class="seats__body-submit">
+                        <div class="seats__body-submit-btn">
+                            <button type="submit">Оформить заказ</button>
```
Comment (A-Ostrovnyy) [inline submissions/Roophee/html-movie-seat-booking/index.html:420]: Add styles on `:hover`, ':focus', ':active' states.

File: submissions/Roophee/html-movie-seat-booking/index.html [hunk 6]
```diff
@@ -0,0 +1,428 @@
...
+                    <a href="https://www.google.com.ua/maps/place/49.994583, 36.233861" target="_ablank">г.Харьков, пл.Театральная, 5 </a>
+                </address>
+                <a href="tel:+380669595621"> +380669595621</a>
+                <br>
+                <a href="mailto:cinema.city@cinema.com">cinema.city@cinema.com</a>
+        </section>
+        <section class="footer__logo">
+            <img class="footer__logo-img" src="img/logo1.jpg" alt="Cinema">
+            <img class="footer__logo-img " src="img/logo3.png" alt="The Mockingbird Cinema">
+            <img class="footer__logo-img " src="img/logo2.png" alt="Little Cinema">
```
Comment (A-Ostrovnyy) [inline submissions/Roophee/html-movie-seat-booking/index.html]: Wrap all images by tag `a`.

File: submissions/Roophee/html-movie-seat-booking/index.html [hunk 7]
```diff
@@ -0,0 +1,428 @@
...
+                            <a href="#" class="header__nav-item">Главная</a>
+                        </li>
+                        <li class="header__nav-item">
+                            <a href="#" class="header__nav-item">О&#160;нас</a></li>
+                        <li  class="header__nav-item">
+                            <a href="#" class="header__nav-item">Сеансы&#160;и&#160;Билеты</a></li>
+                        <li class="header__nav-item">
+                            <a href="#" class="header__nav-item">Контакты</a></li>
+                        <li class="header__nav-item">
+                            <a href="#" class="header__nav-item">Форум</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/Roophee/html-movie-seat-booking/index.html:33]: Increase font-size on mobile.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comment below!

File: submissions/Roophee/html-movie-seat-booking/index.html (hunk 1 above)
Comment (Roophee) [reply submissions/Roophee/html-movie-seat-booking/index.html]: Done. Changed div on button element. Thanks.

File: submissions/Roophee/html-movie-seat-booking/style.scss (hunk 2 above)
Comment (Roophee) [reply submissions/Roophee/html-movie-seat-booking/style.scss]: I desided apply this styles only once on table element and used in this appoach selector for all childs whis tags th and td.
Thanks.

File: submissions/Roophee/html-movie-seat-booking/index.html (hunk 3 above)
Comment (Roophee) [reply submissions/Roophee/html-movie-seat-booking/index.html]: Used button element and  p not needed anymore.

File: submissions/Roophee/html-movie-seat-booking/index.html (hunk 4 above)
Comment (Roophee) [reply submissions/Roophee/html-movie-seat-booking/index.html:145]: Done.

File: submissions/Roophee/html-movie-seat-booking/index.html (hunk 5 above)
Comment (Roophee) [reply submissions/Roophee/html-movie-seat-booking/index.html:420]: Done. Thanks

File: submissions/Roophee/html-movie-seat-booking/index.html (hunk 6 above)
Comment (Roophee) [reply submissions/Roophee/html-movie-seat-booking/index.html]: Done. Thanks.

File: submissions/Roophee/html-movie-seat-booking/index.html (hunk 7 above)
Comment (Roophee) [reply submissions/Roophee/html-movie-seat-booking/index.html:33]: Done. Please, tell me how it looks.
Thanks.

**Comment by Roophee** [comment]
I added in `<form>`  section with date in lines 327-340 and used there `<time>` tag and `<datetime> `attribute. @A-Ostrovnyy  Is it wright usage?
Thanks.

File: submissions/Roophee/html-movie-seat-booking/index.html (hunk 7 above)
Comment (A-Ostrovnyy) [reply submissions/Roophee/html-movie-seat-booking/index.html:33]: Looks great for me.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**